box-share-sdk/src/main/java/com/box/androidsdk/share/vm/DataWrapper.java

```diff
@@ -0,0 +1,55 @@
+package com.box.androidsdk.share.vm;
+
+import androidx.annotation.StringRes;
+
+/**
+ * A data wrapper to make sure LiveData changes can take network responses into account
+ * @param <T> the data type of the item in the response that will be returned
+ */
+public class DataWrapper<T> {
```

Comment:
We thought about this for a while for a less generic name - suggest PresenterData.

Reply:
I will make the change.